lib/daru/vector.rb

```diff
@@ -1164,6 +1164,16 @@ def method_missing(name, *args, &block)
       end
     end
 
+    def cut partitions
+      partitions = partitions.to_a
```

Comment:
It is residing in `vector.rb` because it converts from a numeric type vector by categorizing it into categories.